lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -605,8 +605,20 @@ export class PreviewWeb<TFramework extends AnyFramework> {
 
         if (forceRemount && playFunction) {
           this.disableKeyListeners = true;
-          await runPhase('playing', () => playFunction(renderContext.storyContext));
-          await runPhase('played');
+          await runPhase('playing', async () => {
+            try {
+              await playFunction(renderContext.storyContext);
+              await runPhase('played');
+            } catch (error) {
+              if (error !== IGNORED_EXCEPTION) {
```

Comment:
An ignored exception is one that was picked up by the instrumenter? Is there any reason to treat them differently at this level now?